### PR TITLE
Feat: Cu-865c5hh4w: UI components login box

### PR DESCRIPTION
**Pull request by AndresOL2001**
# Fixes ticket [#t865c5hh4w](https://app.clickup.com/t/865c5hh4w)

## Checklist 
- [x] Passes eslint linter 
- [ ] Passes backend integration tests 
- [ ] Added tests for this change 
- [x] Added documentation for this change 
- [x] Removed unused packages  

## Changes proposed in this pull request
Login Component box added

## Reviewers
- @aviada-rob 
- @eduardoMtzCh 
- @jorgelabs 
- @aviada-andreazayas

### CODE REVIEW

**Comment by aviada-fgp** [comment]
Task linked: [CU-865c5hh4w UI Components Login Box](https://app.clickup.com/t/865c5hh4w)

**Review by aviada-andreazayas — CHANGES_REQUESTED** [review]
Remove the tests for the other components that are not in the Login Box and add them on their own PR's please 😄

**Comment by AndresOL2001** [comment]
Ready

**Review by aviada-andreazayas — CHANGES_REQUESTED** [review]

File: src/app/components/Organisms/Login/Login.mocks.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { ILogin } from "./Login";
+
+const base: ILogin = {
+  AlertError: true,
```
[the remark this answers is not in the document]
Comment (AndresOL2001) [reply src/app/components/Organisms/Login/Login.mocks.ts]: Done

File: src/app/components/Organisms/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import TextboxInput from "../../../components/Atoms/TextboxInput/TextboxInput";
+import BaseButton from "../../Atoms/Buttons/BaseButton/BaseButton";
+import Panel from "../../Atoms/Panel/Panel";
+import Alert from "../../Molecules/Alerts/Alert";
+export interface ILogin {
+  AlertError: boolean;
```
[the remark this answers is not in the document]
Comment (AndresOL2001) [reply src/app/components/Organisms/Login/Login.tsx]: Done

**Review by aviada-andreazayas — APPROVED** [review]

**Pull request merged**